specification/electronic-prescription-service-api.yaml

```diff
@@ -314,7 +314,7 @@ paths:
             examples:
               group-identifier:
                 value:
-                  $ref: components/examples/secondary-care/community/acute/nominated-pharmacy/clinical-practitioner/1-Release-Request-200_OK.json
+                  $ref: components/examples/secondary-care/community/acute/nominated-pharmacy/clinical-practitioner/1-SKIP-Release-Request-200_OK.json
```

Comment:
Does this mean we're not testing this request?

Reply:
Yeah, it's a request for an interaction we're not doing yet